src/mozillavpn.h

```diff
@@ -233,7 +236,7 @@ class MozillaVPN final : public QObject {
 
   [[nodiscard]] bool setServerList(const QByteArray& serverData);
 
-  void reset(bool forceInitialState);
+  Q_INVOKABLE void reset(bool forceInitialState);
```

Comment:
why Q_INVOKABLE? I don't see where this is used.

Reply:
It's used for the "we couldn't auth screen"